tests/kas_with_instance.spec.ts

```diff
@@ -67,7 +67,7 @@ const filterByName = async function (page, name, skipClick = false) {
 // test_3kas.py test_kas_kafka_filter_by_name
 test('test instances can be filtered by name', async ({ page }) => {
   await filterByName(page, 'test');
-  await expect(page.getByText(testInstanceName)).toBeTruthy();
```

Comment:
revert all the current changes in this file, please.

Reply:
![image](https://user-images.githubusercontent.com/100594859/203090709-3eb3fc22-d32a-4bc2-a871-97689e976257.png)

Reply:
as above